packages/aws-cdk-lib/aws-route53/lib/geo-location.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Routing based on geographical location.
+ */
+export class GeoLocation {
+  /**
+   * Geolocation resource record based on continent code.
+   * @param continentCode Code of the continent. Valid codes are AF (Africa), AN (Antarctica), AS (Asia),
+   * EU (Europe), OC (Oceania), NA (North America), SA (South America).
+   * @returns Continent-based geolocation record
+   */
+  public static continent(continentCode: string) {
+    return new GeoLocation(continentCode, undefined, undefined);
+  }
```

Comment:
Since this has a very small list of acceptable values (and new continents are unlikely), is there value in this being an enum instead?

```ts
export enum Continent {
  AFRICA = "AF",
  ANTARCTICA = "AN",
  ASIA = "AS",
  EUROPE = "EU",
  OCEANIA = "OC",
  NORTH_AMERICA = "NA",
  SOUTH_AMERICA = "SA"
}
```

That way users can do `route53.GeoLocation.continent(route53.Continent.ASIA)` instead of needing to know the code (and perhaps have autocomplete/suggestions from an IDE/tool)?

It might be a bit overly verbose to do that though, especially since enums aren't really the easiest option for the remaining values.

Reply:
Yes, this makes sense. The enum is easier to use. It's not consistent anymore because country uses a string and continent uses an enum. However, I like the autocompletion.